.github/workflows/prbot.yml

```diff
@@ -46,20 +46,77 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        spec:
+        containers:
           [
```

Comment:
nit: can we use a js expression to generate the range of numbers e.g. `${{ [...Array(PARALLEL_JOBS).keys()].splice(1) }}`

Reply:
Yes could have, although we would have to put it in a runner before and then use the output of this pre runner just to set a static array... I felt this was not really the best, especially because well. We already use heaps and heaps of - runners with that